Harvest.Web/ClientApp/src/Requests/RequestContainer.tsx

```diff
@@ -108,8 +108,11 @@ export const RequestContainer = () => {
   var isFilledIn = useMemo(() => {
     return (
       project.start !== undefined &&
+      project.start !== null &&
       project.end !== undefined &&
-      project.crop !== undefined
+      project.end !== null &&
+      project.crop !== undefined &&
+      project.crop !== ""
```

Comment:
Probably better to just use falsey values here.  Usually strict equals is nice but if there are this many options i'd be easier to just say `project.start && project.end && project.crop` and be done with it.